core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.ts

```diff
@@ -231,7 +231,11 @@ export class EnvironmentDetailComponent implements OnInit {
               return asset.asset_type === 'swagger_url';
             });
             if (foundAsset) {
-              return window.open(environment.urls['swagger_editor'] + foundAsset.provider_id);
+              if(foundAsset.provider == 'azure'){
```

Comment:
why this difference?

Reply:
Agreed, this experience should be consistent irrespective of the provider.

Reply:
reverted back the changes